### DISCUSSION TITLE
скрипт авторизации

### DISCUSSION THREAD

**Discussion by nglmq** [xob0t/MM-tools#4] [Ideas]
Можно ли реализовать скрипт авторизации для мегамаркета?

**Comment by xob0t** [comment]
Что имеется ввиду под "авторизацией"?

**Reply by nglmq** [reply]
Имею ввиду вход в аккаунт с помощью скрипта

**Reply by xob0t** [reply]
> Имею ввиду вход в аккаунт с помощью скрипта

нужно контентнее, как предполагается оно должно работать?
Вот прямо по шагам.

**Reply by nglmq** [reply]
Пытаюсь реализовать скрипт, который будет покупать товар с мегамаркета. Пытался реализовать вход исходя тз fetch/xhr запросов, но пока не получилось. Вход с помощью смс кода или сберАйди реализовать не пробовал. 
Самый легкий путь логина, наверное, через куки (ecom_token, sbermegamarket_token), но не получается, поэтому решил спросить у вас)

**Reply by xob0t** [reply]
> покупать товар с мегамаркета

юзерскрипты для таких задач это плохая идея
лучше использовать selenium/bas/zennoposter

**Reply by nglmq** [reply]
можно же обращаться к апи

**Reply by xob0t** [reply]
если полностью через api, то и браузер не нужен
можно сделать в python/node или где угодно
а cookie экспортированные cookieEditor'ом в json можно импортировать вот так
https://gist.github.com/xob0t/7b81d569ca76a9007f28fe1c9eae0a8b

**Reply by nglmq** [reply]
да, спасибо, пробую писать на нод
